crates/bevy_winit/src/lib.rs

```diff
@@ -262,6 +264,17 @@ struct WinitAppRunnerState {
     last_update: Instant,
     /// The time the next update is scheduled to start.
     scheduled_update: Option<Instant>,
+    /// Number of "forced" updates to trigger on application start
+    start_forced_updates: u32,
```

Comment:
`initial_forced_updates` or `startup_forced_updates` would be clearer here, as `start` can be a verb.